### PR TITLE
Mint Signup Page Clone

### PR DESCRIPTION
**Pull request by OlawaleJoseph**
# This pull request features a cloned signup page of [MInt](http://accounts.intuit.com/signup.html).
 
## This branch features the following

- Creating a form
- Using different input types
- Using form attributes such as method, action.
- Writing modularized CSS code
- Using of Linters and Sticklers

### CODE REVIEW

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @OlawaleJoseph and @mohapakram   :handshake: 

### Good job so far :clap:  your project is looking good :medal_sports: 

### there are still some changes to get the project approved :point_down:
- You have stickler files in your repository but I don't see any stickler checks in the pull request, please go to https://stickler-ci.com/ and enable stickler.
- In your Readme file there is non link to the live demo, please use http://raw.githack.com/ and update your readme file.
- The project should have a sign-in button:
##### Sign In Button
- [ ] It has a Sign In button at the top right corner ([link to page](https://accounts.intuit.com/signup.html))

Original:
Screenshot from 2020-06-03 13-12-40
Your project:
Screenshot from 2020-06-03 13-13-53
- The page should have the intuit logo and products:
##### Intuit logo
- [ ] It shows the Intuit logo above the form ([link to page](https://accounts.intuit.com/signup.html))
- [ ] The intuit logo is an embedded image that is horizontally centered ([link to page](https://accounts.intuit.com/signup.html))

##### Intuit products
- [ ] The products row is under the Intuit logo and above the form ([link to page](https://accounts.intuit.com/signup.html))
- [ ] It shows the Mint logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))
- [ ] It shows the Quickbooks logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))
- [ ] It shows the Turbotax logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))

Original:
Screenshot from 2020-06-03 13-15-02
Your Project:
Screenshot from 2020-06-03 13-16-30
- The bottom of the form is different from the original:
###### Bottom
- [ ] It has a disclaimer text that matches the context and style of the original page ([link to page](https://accounts.intuit.com/signup.html))
Original
Screenshot from 2020-06-03 13-20-07
Your Project:
Screenshot from 2020-06-03 13-20-17
- Having a footer is required:
##### Footer
- [ ] It contains copyright and address information ([link to page](https://accounts.intuit.com/signup.html))
Your project:
Screenshot from 2020-06-03 13-22-05
Original:
Screenshot from 2020-06-03 13-21-57
### Check the comments for more details.
### Please implement the required changes and ask for another code review.

**Comment by voscarmv** [comment]
# Pull Request under the review by Oscar Mier (TSE) 👨‍💻

File: style.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+/* Body */
+body {
+  background-color: #eceef1;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 16px;
+}
+
+/* Form */
```
Comment (voscarmv) [inline style.css]: Please remove **all** comments from your CSS. This will make your code look cleaner ✨

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Great work implementing the requested changes team! 🔨 👨‍💻 🔧 
## However, additional changes are required... ⚙️ ⚙️ 

Please use this HTML validator before submitting any code review:

https://validator.w3.org/nu/

When running this validator, the following error was found:

- Please fix this by removing the unnecesary `type` attribute from your `<script>` tag.
image

**[OPTIONAL]** Use semantic `<nav>`, `<main>` and `<footer>` tags to section your `index.html` page! This will make your page easier to process by [web spiders](https://en.wikipedia.org/wiki/Web_crawler) 🕸️ 🕷️  and it will help robots improve your `SEO` 🤖!

### Before re-submitting, please also check the comments I've left on your code for additional changes! 🗨️ 🧑‍💻 

# Happy coding! 😄 👍

**Comment by mohapakram** [comment]
@voscarmv Hi, thank you for the feedback.
we don't have any scripts in our HTML but the service we use to serve the live demo may add some scripts and we have no control over it so please when checking our HTML for validation use our original HTML file (index.html) in the repo and not the live demo link.

File: style.css (hunk 1 above)
Comment (mohapakram) [reply style.css]: In the project description, there is a link to an article talking about CSS best practices and they mentioned using a CSS Table of Contents as a best practice. 

check it right here: https://speckyboy.com/good-bad-css-practices/

Thank you for the review.

**Comment by tirthajyoti-ghosh** [comment]
# Pull Request under review by T. Ghosh (TSE)

**Review by tirthajyoti-ghosh — APPROVED** [review]
# Status: Approved✔

Great job on applying the changes. It's 👍 from me.
However, consider making these optional changes.

Check the comments.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+          <h1 class="ver-margin">Create an Intuit account</h1>
+          <div id="form-des" class="ver-margin">
+            <p>One account for everything Intuit, including Mint.</p>
+            <a href="#">Learn more</a>
+          </div>
+        </div>
+        <form action="#" method="post">
+          <div class="input-container">
+            <label for="email">Email address</label>
+            <input type="text" name="email" id="email" required />
```
Comment (tirthajyoti-ghosh) [inline index.html]: [OPTIONAL] Make your input `type` be `email`. Eg., `<input type="email"...` for email inputs.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+          </div>
+          <div class="input-container">
+            <label for="phone">Phone (recommended)</label>
+            <input type="number" name="phone" id="phone" required />
```
Comment (tirthajyoti-ghosh) [inline index.html]: [OPTIONAL] Make your input `type` be `tel`. Eg., <input type="tel"... for phone number inputs.

**Review by tirthajyoti-ghosh — COMMENTED** [review]
Consider applying these small and optional changes before merging.

Good luck on your next projects ✨

File: index.html (hunk 2 above)
Comment (OlawaleJoseph) [reply index.html]: Resolved

File: index.html (hunk 3 above)
Comment (OlawaleJoseph) [reply index.html]: Resolved. Thanks for the feedback

**Pull request merged**